.cirrus.yml

```diff
@@ -195,25 +195,26 @@ task:
 task:
   name: macOS
   macos_instance:
-    image: ghcr.io/cirruslabs/macos-sonoma-base:latest
+    image: ghcr.io/cirruslabs/macos-runner:sonoma
   env:
     HAVE_IPV6_LOCALHOST: yes
     USE_SUDO: true
+    CPPFLAGS: -I/opt/homebrew/opt/openssl@3/include
+    LDFLAGS: -L/opt/homebrew/opt/openssl@3/lib
   setup_script:
     - brew install autoconf automake bash libevent libtool openssl pandoc pkg-config postgresql@${PGVERSION}
     - python3 -m venv venv
     - venv/bin/pip install -r requirements.txt
     - echo 'anchor "pgbouncer_test/*"' | sudo tee -a /etc/pf.conf
     - sudo pfctl -f /etc/pf.conf
-  env:
-    CPPFLAGS: -I/opt/homebrew/opt/openssl@3/include
-    LDFLAGS: -L/opt/homebrew/opt/openssl@3/lib
   build_script:
     - ./autogen.sh
     - ./configure --prefix=$HOME/install --enable-werror
     - make -j4
+  env:
+    PATH: /opt/homebrew/opt/postgresql@${PGVERSION}/bin:$PATH
```

Comment:
can this be merged with the other `env` section?

Reply:
I think so. The main reason I added here was that that path exists only after installing the PostgreSQL package.

Reply:
Let's merge them then, because it's a bit confusing that we have two `env` keys in the same object. Based on some quick searching it doesn't seem specified what that means according the yaml spec (it seems like github merges the objects)